src/main/resources/application.properties

```diff
@@ -1,7 +1,10 @@
-spring.datasource.url=${DB_URL_TEST}
-spring.datasource.password=${DB_PASSWORD_TEST}
+#trying this locally!
+#spring.datasource.url=jdbc:postgres://localhost:5432/dex
+#spring.datasource.password=password
+#spring.datasource.username=root
 spring.datasource.driver-class-name=org.postgresql.Driver
```

Comment:
many tutorials tell me to use this application.properties file but i don't see how the application pulls it's values from here, will test another time.